.github/workflows/graalvm.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        graalvm: ['20.3.0.java8', '20.3.0.java11']
+        graalvm: ['21.0.0.java8', '21.0.0.java11']
```

Comment:
Latest version is `21.0.0.2`. Not sure if that version will work with the action we use to download GraalVM